### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by Bharat-Fexle-May2022**

### CODE REVIEW

File: src/classes/PatternCreator.cls [hunk 1]
```diff
@@ -0,0 +1,81 @@
+public with sharing class PatternCreator {
+    public PatternCreator() {
```
Comment (SeemuSaikiaSF) [inline src/classes/PatternCreator.cls:2]: Use it or remove it

File: src/classes/PatternCreator.cls [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+            System.debug(s);
+        }
+        for(Integer l = 1; l <= 5; l++) {
+            String s = ' ';
+            for(Integer m = l; m <= 5; m++) {
+                s = s + ' ';
+            }
+            for(Integer n = 1; n <= l; n++) {
+                s = s + '* ';
+            }
```
Comment (SeemuSaikiaSF) [inline src/classes/PatternCreator.cls:78]: System debug statement is missing ?

Comment (Bharat-Fexle-May2022) [reply src/classes/PatternCreator.cls:75]: system.debug is added

**Review by Bharat-Fexle-May2022 — COMMENTED** [review]
All problems are resolved

**Pull request merged**